lib/gollum/public/gollum/stylesheets/sass/editor.scss

```diff
@@ -0,0 +1,504 @@
+// Wiki editor formatting
+
+@import "_base", "_features", "_breakpoint", "_component";
+
+a {
+  -moz-outline: none !important;
+}
+
+.jaws {
+  // JAWS should see it, but you can't
+  display: block;
+  height: 1px;
+  left: -5000px;
+  overflow: hidden;
+  position: absolute;
+  top: -5000px;
+  width: 1px;
+}
+
+.disabled {
+  color: lightgray;
+}
+
+#gollum-editor-body-ace {
```

Comment:
I just pasted the CSS for the ace editor here. If there's a better way of incorporating this in the new SCSS I'd be happy to hear about it.